tests/istio.mk

```diff
@@ -100,8 +100,10 @@ e2e_simple_noauth_run: out_dir
 	--v1alpha1=false --v1alpha3=true --egress=false --ingress=false \
 	--rbac_enable=false --cluster_wide ${E2E_ARGS} ${T} ${EXTRA_E2E_ARGS} ${CAPTURE_LOG}
 
-e2e_mixer_run: out_dir
-	go test -v -timeout 20m ./tests/e2e/tests/mixer -args ${E2E_ARGS} ${EXTRA_E2E_ARGS}
+e2e_mixer_run: out_dir generate_yaml-envoyv2_transition_loadbalancer_ingressgateway
```

Comment:
generate_yaml-envoyv2_transition_loadbalancer_ingressgateway is not needed here since it is already done in "e2e_mixer"